wings/market/ddex_market.pyx

```diff
@@ -271,6 +272,10 @@ cdef class DDEXMarket(MarketBase):
                and not math.isnan(self._gas_fee_weth) \
                and not math.isnan(self._gas_fee_usd)
 
+    @property
+    def name(self) -> str:
+        return "DDEX"
+
```

Comment:
Is there a reason for us to change the Market names to uppercase?  If not, it's better to keep it consistent since the CLI uses all lowercase